CodeEdit/NavigatorSidebar/NavigatorSidebarItem.swift

```diff
@@ -3,55 +3,255 @@
 //  CodeEdit
 //
 //  Created by Lukas Pistrol on 17.03.22.
+//  Context Menu by Nanashi Li on 20.03.22.
 //
 
 import SwiftUI
 import WorkspaceClient
 import CodeFile
 
 struct NavigatorSidebarItem: View {
-	@AppStorage(FileIconStyle.storageKey) var iconStyle: FileIconStyle = .default
-
-	var item: WorkspaceClient.FileItem
+    @AppStorage(FileIconStyle.storageKey) var iconStyle: FileIconStyle = .default
+    
+    var item: WorkspaceClient.FileItem
     @ObservedObject var workspace: WorkspaceDocument
     var windowController: NSWindowController
-	@State var isExpanded: Bool = false
-
-	var body: some View {
-		if item.children == nil {
-			sidebarFileItem(item)
-				.id(item.id)
-		} else {
-			sidebarFolderItem(item)
-				.id(item.id)
-		}
-	}
-
+    @State var isExpanded: Bool = false
+    
+    let fileManger = FileManager.default
+    
+    // maximum number of views in a container is exceeded (in SwiftUI). The max = 10
+    var body: some View {
+        if item.children == nil {
+            sidebarFileItem(item)
+                .id(item.id)
+                .contextMenu{
+                    Button("Show in Finder", action: {
+                        showInFinder()
+                    })
+                    
+                    Group{
+                        Divider()
+                        
+                        Button("Open in Tab", action: {
+                            print("Open in Tab")
```

Comment:
please remove `print` statements as stated in the [contribution guide](https://github.com/CodeEditApp/CodeEdit/blob/main/CONTRIBUTING.md)

replace with:
```swift
// Open in Tab
```

and add a todo above the whole block that implementation of some actions is still missing